docs/source/installation/troubleshooting.rst

```diff
@@ -182,3 +182,108 @@ operating system's documentation to find out where you can get it and how you
 have to install it.
 
 If you are unable to solve your problem, check out the `dvisvgm FAQ <https://dvisvgm.de/FAQ/>`_.
+
+(Windows) ``Python is not recognized as an internal or external command, operable program or batch file.``
+--------------------------------------------------------------------------------------------------------
+
+1. Open System Properties (Right click Computer in the start menu, or use the keyboard shortcut Win+Pause)
```

Comment:
Could you also format certain key phrases as inline code, as was done in the stack exchange answer? Also, copying the text directly from that answer might technically be a violation of copyright.

Reply:
Done and Added as small reference notice